scripts/run-eunomia-locally.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```

Comment:
Has this been tested on both Mac and Linux?

Reply:
Yes, this script and `run-eunomia-locally.sh`. They both work